guides/common/assembly_searching-and-bookmarking.adoc

```diff
@@ -2,20 +2,10 @@
 
 include::modules/con_searching-and-bookmarking.adoc[]
 
-include::modules/con_building-search-queries.adoc[leveloffset=+1]
-
-include::modules/con_query-syntax.adoc[leveloffset=+2]
-
-include::modules/ref_query-operators.adoc[leveloffset=+2]
-
-include::modules/ref_query-values.adoc[leveloffset=+2]
+include::assembly_building-search-queries.adoc[leveloffset=+1]
 
 include::modules/con_using-free-text-search.adoc[leveloffset=+1]
 
-include::modules/con_managing-bookmarks.adoc[leveloffset=+1]
-
-include::modules/proc_creating-bookmarks.adoc[leveloffset=+2]
-
-include::modules/proc_deleting-bookmarks.adoc[leveloffset=+2]
+include::assembly_managing-bookmarks.adoc[leveloffset=+1]
```

Comment:
same as above.

Reply:
I dropped the assembly and included the modules directly in the top-level assembly.